setup.cfg

```diff
@@ -1,25 +1,2 @@
-[isort]
-src_paths = feast,tests
-multi_line_output=3
-include_trailing_comma=True
-force_grid_wrap=0
-use_parentheses=True
-line_length=88
-skip=feast/protos,feast/embedded_go/lib
-known_first_party=feast,feast_serving_server,feast_core_server
-default_section=THIRDPARTY
-
-[flake8]
-ignore = E203, E266, E501, W503
-max-line-length = 88
-max-complexity = 20
-select = B,C,E,F,W,T4
-exclude = .git,__pycache__,docs/conf.py,dist,feast/protos,feast/embedded_go/lib
-
-[mypy]
-files=feast,tests
-ignore_missing_imports=true
-exclude=feast/embedded_go/lib
-
 [bdist_wheel]
```

Comment:
(optional) can we somehow move this config to pyproject.toml? so that we can eliminate the `setup.cfg` file

Reply:
I don't feel confident to change this one, even if I agree with your request to minimize the configuration files.
The `wheels` docs only mention the `setup.cfg` as the configuration option, so I'm not sure which of the different alternatives Google offers to trust and how to validate them later.
Can we have a dedicated ticket instead?

Reply:
@sudohainguyen rather, I would move the formatting configuration from `/pyproject.tom`l to `/sdk/python/pyproject.toml`, that seems its natural location, looking at the `Makefile`. WDYT?

Reply:
I prefer keeping as where it is now. I believe in the future we will eventually deprecate `setup.py` to use `pyproject.toml` for publishing, according to [PEP 621](https://peps.python.org/pep-0621/)
So I don't wanna mess up that flow 😄 

back to `setup.cfg`, let's leave it there for the further PR